lib/releaselib.sh

```diff
@@ -397,31 +397,38 @@ release::set_release_version () {
 }
 
 ###############################################################################
-# Create GCS bucket for releasing Kube if necessary. Ensure that the default
+# Create GCS bucket for publishing. Ensure that the default
 # ACL allows public reading of artifacts.
+#
 # @param bucket - The gs release bucket name
 # @return 1 if bucket can't be made
 release::gcs::ensure_release_bucket() {
   local bucket=$1
-  local current_defacl
-  local new_acl_file
+  local tempfile=/tmp/$PROG-gcs-write.$$
 
   if ! $GSUTIL ls "gs://$bucket" >/dev/null 2>&1 ; then
     logecho -n "Creating Google Cloud Storage bucket $bucket: "
     logrun -s $GSUTIL mb -p "$GCLOUD_PROJECT" "gs://$bucket" || return 1
-    logecho -n "Adding public-read default ACL on bucket $bucket: "
+  fi
+
+  # Bootstrap security release flow by making bucket visibility private
+  # in that case
+  if [[ "$PARENT_BRANCH" =~ release- ]]; then
```

Comment:
it might be more obvious if this was passed as an option to `ensure_release_bucket` (something like `make_public` being set to false on a security release), but this is fine for now.